src/components/common/ShortBech32.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="short-bech32">
```

Comment:
suggestion: rename to "Address"

Reply:
there is bech32 specific logic here though - so maybe `bech32address` or something?

not sure if this is necessary for this PR

Reply:
I meant renaming the whole component as it is not only "Short" anymore.

Reply:
yep, agreed. but i think it should remain more specific than `address` because of the bech32 logic

Reply:
Both fair points. Keeping it as Bech32 works when looking at the templates code. It reads like:

```
<Bech32 :address="value" />
```


Though perhaps

`<Address :bech32="…" />` or `<Address :value="…" />` is better?

Reply:
yeah good illustration. whatever you think is best.